spec/components/personal_details_review_component_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe PersonalDetailsReviewComponent do
+  it 'renders SummaryCardComponent with valid personal details' do
+    application_form = create(:application_form, :completed_application_form)
+    result = render_inline(PersonalDetailsReviewComponent, application_form: application_form)
+
+    expect(result.css('.app-summary-card')).to be_any
```

Comment:
`be_any` is a mysterious matcher! Asserting on a CSS class feels quite gritty. Could we be more trusting — maybe put a string into the `application_form` and see it crop up in the presented HTML?